Jenkinsfile

```diff
@@ -1,4 +1,4 @@
-@Library('cicd-lib@0.3') _
+@Library('cicd-lib@feature/INGK-885-automate-the-release-process') _
```

Comment:
Shall we create an issue to remember to update this version after the lib is released?

Reply:
Just released 0.11, it can be updated

Reply:
Done